### PR TITLE
Moved to Maven project. Ensure working with 14.0.2

### PR DESCRIPTION
**Pull request by meriouma**

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -22,11 +22,12 @@
...
       </profile>
     </annotationProcessing>
     <bytecodeTargetLevel target="1.6">
+      <module name="gwtp-idea" target="1.7" />
       <module name="plugin-templates" target="1.6" />
     </bytecodeTargetLevel>
   </component>
-</project>
-
+</project>
```
Comment (christiangoudreau) [inline .idea/compiler.xml:33]: missing nl at eof

File: .idea/encodings.xml [hunk 2]
```diff
@@ -29,10 +29,5 @@
...
     <file url="file://$PROJECT_DIR$/lib/tagsoup-1.2.1.jar" charset="UTF-8" />
     <file url="file://$PROJECT_DIR$/lib/velocity-1.5.jar" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/src/com/arcbees/plugin/idea/domain/Archetype.java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/src/com/arcbees/plugin/idea/domain/ArchetypeCollection.java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/src/com/arcbees/plugin/idea/domain/Category.java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/src/com/arcbees/plugin/idea/domain/Tag.java" charset="UTF-8" />
-    <file url="file://$PROJECT_DIR$/src/com/arcbees/plugin/idea/wizards/createproject/FetchArchetypes.java" charset="UTF-8" />
   </component>
-</project>
+</project>
```
Comment (christiangoudreau) [inline .idea/encodings.xml:8]: missing nl at eof

File: .idea/modules.xml [hunk 3]
```diff
@@ -2,8 +2,7 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
-      <module fileurl="file://$PROJECT_DIR$/gwtp-idea-plugin.iml" filepath="$PROJECT_DIR$/gwtp-idea-plugin.iml" />
+      <module fileurl="file://$PROJECT_DIR$/gwtp-idea.iml" filepath="$PROJECT_DIR$/gwtp-idea.iml" />
     </modules>
   </component>
-</project>
-
+</project>
```
Comment (christiangoudreau) [inline .idea/modules.xml:8]: missing nl at eof

**Comment by christiangoudreau** [comment]
Few nits, then LGTM

File: .idea/modules.xml (hunk 3 above)
Comment (meriouma) [inline .idea/modules.xml:8]: I don't even know why those files are in the repo. I don't think the .idea folder should be there

Comment (christiangoudreau) [inline .idea/modules.xml:8]: I also don't think. Remove what's unneeded :D

Comment (meriouma) [inline .idea/modules.xml:8]: I think I removed it with git, but it doesn't seem to show in the PR. I'll see what happens when I merge, then I'll make sure it's gone

**Pull request closed without merging**